gym_lowcostrobot/envs/lift_cube_env.py

```diff
@@ -109,11 +109,17 @@ def __init__(self, observation_mode="image", action_mode="joint", render_mode=No
             self.rgb_array_renderer = mujoco.Renderer(self.model, height=640, width=640)
 
         # Set additional utils
+        self.done = False # termination flag
+        self.success = False # success flag
         self.threshold_height = 0.5
         self.cube_low = np.array([-0.15, 0.10, 0.015])
         self.cube_high = np.array([0.15, 0.25, 0.015])
+        self.target_low = np.array([-3.14159, -1.5708, -1.48353, -1.91986, -2.96706, -1.74533])
+        self.target_high = np.array([3.14159, 1.22173, 1.74533, 1.91986, 2.96706, 0.0523599])
+        self.q0 = (self.target_high + self.target_low) / 2 # home position
```

Comment:
No, it's not the home (or neutral) position.  The neutral position is 0 in joint space, so it corresponds to `-(target_high + target_low) / (target_high - target_low)` in action space.

Reply:
This is used in 'ee' mode for computing the inverse kinematics, so that the solution is close to the home position. maybe it is a good idea to put it inside the inverse kinematics function?